azure-pipelines.yml

```diff
@@ -17,6 +17,11 @@ pr:
       - v4
 
 steps:
+- task: ComponentGovernanceComponentDetection@0
```

Comment:
Could you remove this before the change? 
This is automatically injected, just debugging purpose during on a branch.

Reply:
Yes will take a note to remove it after the final review